secret.go

```diff
@@ -41,6 +43,22 @@ func ParseConfigArgument(configArg string) (string, error) {
 	return configPath, nil
 }
 
+const (
+	// The rate limiter replenishes tokens in the bucket once every 200 ms
+	// (5 per second) and allows a burst of 10 requests firing faster than
+	// that. If the client keeps making requests at the rapid pace, they
+	// will be slowed down. This allows a client to ask for a bunch of
+	// secrets at the start of a script, and then it slows it down to a
+	// reasonable pace. A single script using more than 25 secrets is
+	// probably a bad idea anyway. The reate limit is here to protect the
+	// Grafana Secrets API from being hammered too hard by bugs in a
+	// script (for example, someone using a secret inside a loop).
+	//
+	// These values can be adjusted as needed.
+	tokenReplenishInterval = 200 * time.Millisecond
+	burstLimit             = 10
```

Comment:
Should we make this configurable via the config and/or the environment so its easier to change without rolling a release of this (and then xk6-sm and then the agent)

Reply:
I have added the option to optionally read the values from the configuration file.